ci/build.sh

```diff
@@ -10,11 +10,11 @@ ci/bootstrap.sh
 
```

Comment:
File permissions on this got rekt. Should be reverted to `100755`

Reply:
good spot, fixed 👍